test/ranking.jl

```diff
@@ -37,3 +37,31 @@ s = ["c", "a", "b", "d", "d", "b", "e", "d"] # s is a vector of strings ordered
 @test tiedrank(s) == tiedrank(x)
 @test tiedrank(x, rev = true) == tiedrank(-x)
 @test tiedrank(x, lt = (x, y) -> isless(y, x)) == tiedrank(-x)
+
+@testset "quantilerank and percentilerank" begin
+
+    pcrk = percentilerank
+    qtrk = quantilerank
+ 
+    @testset "score as number and array" begin        
+        for (method, res1, res2) in [(:rank, 40.0, [40.0, 85.0]), 
+                                    (:strict, 30.0, [30.0, 80.0]),
+                                    (:weak, 50.0, [50.0, 90.0]), 
+                                    (:mean, 40.0, [40.0, 85.0])]
```

Comment:
Can you test a case where `:mean` and `:rank` do not give the same result?

Reply:
I found that these two methods are equivalent. 
Therefore, I preferred to remove the `:rank` argument and keep the `:mean` with the methodology of the cited book. 
Done at commit 11671f4.